nightfall-optimist/src/utils/auth.mjs

```diff
@@ -1,11 +1,5 @@
-import config from 'config';
-import crypto from 'crypto';
-
-const environment = config.ENVIRONMENTS[process.env.ENVIRONMENT] || config.ENVIRONMENTS.localhost;
-
 export default function auth(req, res, next) {
-  const token = req.headers['x-app-token'];
-  if (token === crypto.createHash('sha256').update(environment.PROPOSER_KEY).digest('hex'))
-    return next();
+  const token = req.headers['X-APP-TOKEN'];
+  if (token === process.env.AUTH_TOKEN) return next();
```

Comment:
Should we take this from config?

Reply:
no, if we maintain it in the environments, we need to call it from there.